src/lib.rs

```diff
@@ -484,22 +442,21 @@ pub fn instantiate_module(store: &mut Store, module: ast::Module, extern_vals: &
 		elem_offsets.push(offset);
 
 		let table_size = {
-			if (elem.index as usize) >= imported_tables.len() {
-				module.tables[(elem.index as usize) - imported_tables.len()].type_.limits.min as usize
-			} else {
+			let is_imported = (elem.index as usize) < imported_tables.len();
```

Comment:
Why?

Reply:
Without any intermediate names nor comments, this size computation was not obvious